### PR TITLE
Changed configuration to have autoClose be configured per category

### LINKED ISSUE #120: Allow support room categories to not auto-close
This has been an issue with especially support tickets on bugs. We might want to have the option to configure the auto-close timer on each category rather than on all categories at once.

### PR DESCRIPTION
**Pull request by McJeffr**
Addresses #120.

### CODE REVIEW

File: src/Command/Support.js [hunk 1]
```diff
@@ -124,8 +124,9 @@ module.exports = Command.extend({
             })
             .catch(() => {
                 /* If something went wrong with room creation, we delete it (if it was created) and notify the user */
-                supportChannel.delete().catch(() => {
-                });
+                if (supportChannel) {
```
Comment (McJeffr) [inline src/Command/Support.js:128]: This threw an error due to a misconfiguration on my side, where a channel was never created to begin with, so it couldn't be deleted. Figured I might as well address this whilst we are at it.

File: src/Utility/TrickcordTreat.js [hunk 2]
```diff
@@ -28,8 +28,6 @@ module.exports = BotModule.extend({
         this.timeout = typeof this.config.trickcordTreatDeleteTime != 'undefined' ?
             this.config.trickcordTreatDeleteTime :
             ONE_MINUTE_IN_MS;
-
-        console.log(this.config.trickcordTreatRoom);
```
Comment (McJeffr) [inline src/Utility/TrickcordTreat.js:32]: Tsk tsk tsk

**Review by JamsheedMistri — DISMISSED** [review]
Have you tested? LGTM

**Comment by McJeffr** [comment]
> Have you tested? LGTM

Yup, it seemed to work. But I'll add the other small functionality with pingable roles to it, so I'll put this on hold for a sec.

File: config/default.js [hunk 3]
```diff
@@ -109,28 +109,39 @@ module.exports = {
...
+                ],
+                logRoom: '748963030888349736',
+                autoClose: {
+                    warning: ONE_DAY_IN_MS,
+                    closing: 2 * ONE_DAY_IN_MS
+                }
+            },
+            'bug': {
+                description: "Open a support channel to report a bug to the development team.",
+                roles: [{id: '223821346763833344', ping: true}],
```
Comment (JamsheedMistri) [inline config/default.js]: Can you please change this? `223821346763833344` (Admin) should be included but not pinged. `815871997871128576` (Buganeer) should be included and pinged.

File: src/Utility/TrickcordTreat.js (hunk 2 above)
Comment (navarr) [reply src/Utility/TrickcordTreat.js:32]: 👀

**Review by navarr — DISMISSED** [review]

File: config/default.js (hunk 3 above)
Comment (McJeffr) [reply config/default.js]: Done

**Review by JamsheedMistri — APPROVED** [review]

**Pull request merged**